containerd-shim-v2/wait.go

```diff
@@ -126,3 +128,26 @@ func watchSandbox(s *service) {
 	// Existing container/exec will be cleaned up by its waiters.
 	// No need to send async events here.
 }
+
+func watchOOMEvents(ctx context.Context, s *service) {
+	if s.sandbox == nil {
+		return
+	}
+
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		default:
+			containerID, err := s.sandbox.GetOOMEvent()
+			if err != nil {
+				logrus.WithError(err).Warn("failed to get oom event from sandbox")
```

Comment:
I would spell "OOM" instead of "oom" in the log

Reply:
Thanks! Fixed up.